PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+#### Description of changes made
```

Comment:
This is nice,...!!

Reply:
Maybe you should add a paragraph, where we can put the link to the [gitKraten](https://www.gitkraken.com/) card.